### PR TITLE
Alex/#855 add process type to UI

### PR DESCRIPTION
**Pull request by energinet-ajw**
Added missing column (process type) to batch table.

## References

* #855

### CODE REVIEW

File: source/dotnet/Services/DatabaseMigration/Scripts/202302091300_Add_ProcessType_column_To_Batch.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+﻿ALTER TABLE Batch
```
Comment (lasrinnil) [inline source/dotnet/Services/DatabaseMigration/Scripts/202302091300_Add_ProcessType_column_To_Batch.sql:1]: default for existing batches

File: source/dotnet/Services/Infrastructure/Persistence/Batches/BatchEntityConfiguration.cs [hunk 2]
```diff
@@ -52,5 +52,7 @@ public void Configure(EntityTypeBuilder<Batch> builder)
             .HasConversion(
                 l => JsonSerializer.Serialize(l.Select(code => code.Code), (JsonSerializerOptions?)null),
                 s => JsonSerializer.Deserialize<List<string>>(s, (JsonSerializerOptions?)null)!.Select(code => new GridAreaCode(code)).ToList());
+
+        builder.Property(b => b.ProcessType);
```
Comment (lasrinnil) [inline source/dotnet/Services/Infrastructure/Persistence/Batches/BatchEntityConfiguration.cs:56]: nit: move to other propertys

Comment (energinet-ajw) [reply source/dotnet/Services/Infrastructure/Persistence/Batches/BatchEntityConfiguration.cs]: Done.

File: source/dotnet/Services/DatabaseMigration/Scripts/202302091300_Add_ProcessType_column_To_Batch.sql (hunk 1 above)
Comment (energinet-ajw) [reply source/dotnet/Services/DatabaseMigration/Scripts/202302091300_Add_ProcessType_column_To_Batch.sql:1]: Added default value (0).

**Review by lasrinnil — APPROVED** [review]

**Pull request merged**